tests/cli/test_cli.py

```diff
@@ -289,6 +290,21 @@ def reset_dr_db(dag_id):
 
             reset_dr_db(dag_id)
 
+    def test_kubernetes_migrate_to_pod_template_file(self):
+
+        from airflow.kubernetes.pod_generator import PodGenerator
+        with tempfile.TemporaryDirectory("airflow_dry_run_test/") as directory:
+            d = directory
+            print(d)
+            cli.kubernetes_migrate_to_pod_template_file(self.parser.parse_args(
+                ['kubernetes_migrate_to_pod_template_file', '-o', directory]))
```

Comment:
```suggestion
                ['generate_pod_template', '-o', directory]))
```
I know most people will copy paste but... maybe we can use something simpler? 😄